server/views/pages/createGoals/index.njk

```diff
@@ -28,11 +19,10 @@
         You can add the steps to achieve the goal on the next page.
       </p>
 
-      <form class="form" method="post" novalidate="">
+      <form class="form" method="post" novalidate="" action="create">
         <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
-        <input type="hidden" name="prisonNumber" value="{{ form.prisonNumber }}" />
 
-        {% for goals in form.goals %}
+        {% for goals in formValues.goals or range(1) %}
```

Comment:
Oooh, `or range(1)` .... I take it that's how we can ensure we iterate this loop at least once, rather than adding an empty goal to the array in the form in the controller as we do currently?
Thats neat; I like that 👍